docs/querying/sql-functions.md

```diff
@@ -1227,35 +1269,146 @@ Returns a number for each output row of a groupBy query, indicating whether the
 
 ## HLL_SKETCH_ESTIMATE
 
-`HLL_SKETCH_ESTIMATE(expr, [round])`
+Returns the distinct count estimate from an HLL sketch. The `expr` argument must return an HLL sketch. The optional `round` boolean argument rounds the estimate if set to true, with a default of false.
 
-**Function type:** [Scalar, sketch](sql-scalar.md#sketch-functions)
+* **Syntax:** `HLL_SKETCH_ESTIMATE(expr, [round])`
+* **Function type:** Scalar, sketch
 
-Returns the distinct count estimate from an HLL sketch.
+
+<details><summary>Example</summary>
+
+The following example estimates the number of unique tail numbers in the `flight-carriers` datasource.
+
+```sql
+SELECT
+  HLL_SKETCH_ESTIMATE(DS_HLL("Tail_Number"), False) AS "estimate"
+FROM "flight-carriers"
+```
+
+Returns the following:
+
+| `estimate` | 
+| -- |
+| `4,685.8815405960595` |
+
+</details>
+
+[Learn more](sql-scalar.md#sketch-functions)
 
 ## HLL_SKETCH_ESTIMATE_WITH_ERROR_BOUNDS
 
-`HLL_SKETCH_ESTIMATE_WITH_ERROR_BOUNDS(expr, [numStdDev])`
+Returns the distinct count estimate and error bounds from an HLL sketch. The `expr` argument must return an HLL sketch. The optional round `numStdDev` argument rounds the estimate if set to true, with a default of false.
```

Comment:
```suggestion
Returns a distinct count estimate and error bounds from an HLL sketch. The optional round `numStdDev` argument To round the distinct count estimate, set `numStdDev` to true. Otherwise, `numStdDev` defaults to false.
```